board/safety/safety_subaru.h

```diff
@@ -20,40 +20,56 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const int Brake_Status =  0x13c;
+const int CruiseControl = 0x240;
+const int Throttle = 0x40;
+const int Steering_Torque = 0x119;
+const int Wheel_Speeds = 0x13a;
+
+const int OPENPILOT_STATUS = 0x100;
```

Comment:
What is this?

Reply:
mistake, was for gen2 long with second panda

Reply:
removed